src/spork.cpp

```diff
@@ -90,6 +91,20 @@ void CSporkManager::ProcessSpork(CNode* pfrom, std::string& strCommand, CDataStr
             return;
         }
 
+        // reject old signatures 600 blocks after hard-fork
+        if (spork.nMessVersion != MessageVersion::MESS_VER_HASH) {
+            int nHeight;
+            {
+                LOCK(cs_main);
+                nHeight = chainActive.Height();
+            }
+            if (Params().NewSigsActive(nHeight - 600)) {
```

Comment:
Same here, why `600`?
So far I've found `6000`, `20` and `600`
I think having meaningful constants at one place with some comments would help to understand the underlying logic.
At least it would help me :-)